bin/faucet/src/config.rs

```diff
@@ -44,12 +42,8 @@ impl Display for FaucetConfig {
 impl Default for FaucetConfig {
     fn default() -> Self {
         Self {
-            endpoint: Endpoint {
-                host: "0.0.0.0".to_string(),
-                port: DEFAULT_FAUCET_SERVER_PORT,
-                protocol: Protocol::Http,
-            },
-            node_url: Endpoint::localhost(DEFAULT_NODE_RPC_PORT).to_string(),
+            endpoint: format!("0.0.0.0:{DEFAULT_FAUCET_SERVER_PORT}"),
```

Comment:
```suggestion
            endpoint: format!("http://0.0.0.0:{DEFAULT_FAUCET_SERVER_PORT}"),
```

Reply:
If I'm not mistaken then the http:// prefix would cause failure here
```
let listener = TcpListener::bind(&config.endpoint)
```